src/containers/quiz/quiz-info/QuizInfo.tsx

```diff
@@ -158,4 +167,133 @@ const GradedQuizInfo = ({ points, totalPoints }: GradedQuizInfoProps) => {
   )
 }
 
-export { ActiveQuizInfo, FinishedQuizInfo, UngradedQuizInfo, GradedQuizInfo }
+type StartViewQuizInfoProps = {
+  questionsAmount: number
+  attempts: QuizAttempt
+  timeLimit: QuizTimeLimit
+  usedAttempts: FinishedAttempts
+  handleStartButton: (value: boolean) => void
+}
+const StartViewQuizInfo = ({
+  questionsAmount,
+  attempts,
+  timeLimit,
+  usedAttempts,
+  handleStartButton
+}: StartViewQuizInfoProps) => {
+  const { t } = useTranslation()
+  const [isOpen, setIsOpen] = useState(false)
+
+  const limits = {
+    isNoLimitAttempt: attempts === QuizAttempt.NoLimit,
+    isNoLimitTime: timeLimit === QuizTimeLimit.NoLimit,
+    maxAttempts: parseInt(attempts, 10) || 0
+  }
+  const isHaveAttempts =
+    limits.isNoLimitAttempt || usedAttempts.length < limits.maxAttempts
+
+  const typographyStyle = (subType: number) => {
+    return spliceSx(
+      styles[`subtitle${subType}` as keyof typeof styles],
+      styles.subtitleSize
+    )
+  }
+  const getQuizTimeLimitTitle = (t: TFunction, timeLimit: QuizTimeLimit) => {
+    const timeOption = getQuizTimeLimitFields(t).find(
+      (option) => option.value === timeLimit
+    )
+    return timeOption ? timeOption.title : ''
+  }
+
+  const onStartAttempt = () => {
```

Comment:
All callbacks that passed as a handlers should be named with `handle...`
```suggestion
  const handleStartAttempt = () => {
```